Haskell/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.hi
```

Comment:
You can add this to gitignore in repo root

Reply:
Fixed